File: app/src/main/java/com/example/simonastojanovic/memeapp/model/Response.java
package com.example.simonastojanovic.memeapp.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Response{

	@SerializedName("data")
	private Data data;

	@SerializedName("success")
	@Expose
	private boolean success;

	public void setData(Data data){
		this.data = data;
	}

	public Data getData(){
		return data;
	}

	public void setSuccess(boolean success){
		this.success = success;
	}

	public boolean isSuccess(){
		return success;
	}

	@Override
 	public String toString(){
		return 
			"Response{" + 
			"data = '" + data + '\'' + 
			",success = '" + success + '\'' + 
			"}";
		}
}

File: app/src/main/java/com/example/simonastojanovic/memeapp/model/Response.kt
package com.example.simonastojanovic.memeapp.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Response {

    @SerializedName("data")
    var data: Data? = null

    @SerializedName("success")
    @Expose
    var isSuccess: Boolean = false

    override fun toString(): String {
        return "Response{" +
                "data = '" + data + '\''.toString() +
                ",success = '" + isSuccess + '\''.toString() +
                "}"
    }
}